### ISSUE TITLE
need help

### ISSUE THREAD

**Issue by hartotarigan** [rithvikvibhu/nh-magisk-wifi-firmware#42]
Not work for my phone redmi note 5a

**Comment by rithvikvibhu** [comment]
Which chipset?

**Comment by hartotarigan** [comment]
tp link tl-wn277n

**Comment by rithvikvibhu** [comment]
I can't find that adapter online at all. Link?

**Comment by hartotarigan** [comment]
https://www.google.com/search?q=tl-wn722n+nethunter&safe=strict&prmd=vsin&sxsrf=ACYBGNS0LCgRGo-DwYNvNtlzK6aUX9juLQ:1578912523054&source=lnms&tbm=isch&sa=X&ved=2ahUKEwjgnsXAs4DnAhVlH7cAHUJvAIIQ_AUoA3oECAsQAw&biw=360&bih=560#imgrc=pvIlUey16KKGtM%3A

**Comment by rithvikvibhu** [comment]
That's the 722n, not 277n.
722n is included from v1.

**Comment by hartotarigan** [comment]
![Screenshot_2020-01-13_175702](https://user-images.githubusercontent.com/43029885/72250422-25469680-362e-11ea-8e64-a4b925b799cc.jpg)

**Comment by rithvikvibhu** [comment]
Yes, it is supported from v1 of this module. Have you tried installing and using it? Let me know if there's any problem.

**Comment by hartotarigan** [comment]
I tried and failed. But I'm using version 2

**Comment by rithvikvibhu** [comment]
If it appears in lsusb, but not in iwconfig, then it is likely that the kernel doesn't support this driver.
This module only provides missing firmware systemlessly. A custom kernel is required for the driver. You might find one for your phone on XDA.

**Comment by hartotarigan** [comment]
in xda only 2 kernels are available and both have tried but still failed :'(

**Comment by rithvikvibhu** [comment]
Does it show up in lsusb?
Does it show up in iwconfig?

**Comment by hartotarigan** [comment]
In susb it appears but which appears the name chipset wi-fi

**Comment by cikeZ00** [comment]
Similar issue here, same adapter. It appears in lsusb but not in iwconfig.

Device: OnePlus 5T

The kernel I'm using: https://forum.xda-developers.com/oneplus-5/development/burgerhunter-t3638810

**Comment by cikeZ00** [comment]
If it's a kernel issue, I might as well ask the dev to implement support for it. ¯\_(ツ)_/¯

**Comment by rithvikvibhu** [comment]
I don't know what to say. Can you try https://github.com/aircrack-ng/rtl8188eus?

**Comment by ttlns** [comment]
> tp link tl-wn277n

probably dublicate of https://github.com/rithvikvibhu/nh-magisk-wifi-firmware/issues/76
